apps/web/src/views/Info/components/InfoTables/TransactionsTable.tsx

```diff
@@ -102,10 +102,11 @@ const DataRow: React.FC<React.PropsWithChildren<{ transaction: Transaction }>> =
   const abs1 = Math.abs(transaction.amountToken1)
   const chainName = useChainNameByQuery()
   const { domainName } = useDomainNameForAddress(transaction.sender)
+  const [address0, address1] = [transaction.token0Address, transaction.token1Address].map(isAddress)
```

Comment:
`isAddress` is updated to `safeGetAddress` in https://github.com/pancakeswap/pancake-frontend/pull/7984